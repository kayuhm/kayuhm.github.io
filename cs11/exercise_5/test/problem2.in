18
128
5
306
4371
2264
7695
7696
10000
66672
44449
12352
1000000
48843
48841
15015
10
100
1000
