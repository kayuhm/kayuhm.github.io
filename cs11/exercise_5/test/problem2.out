(2^7)
(5^1)
(2^1)(3^2)(17^1)
(3^1)(31^1)(47^1)
(2^3)(283^1)
(3^4)(5^1)(19^1)
(2^4)(13^1)(37^1)
(2^4)(5^4)
(2^4)(3^2)(463^1)
(44449^1)
(2^6)(193^1)
(2^6)(5^6)
(3^6)(67^1)
(13^2)(17^2)
(3^1)(5^1)(7^1)(11^1)(13^1)
(2^1)(5^1)
(2^2)(5^2)
(2^3)(5^3)
